duckduckgo/privacy-essentials-safari
# optimise downloads

Task/Issue URL: https://app.asana.com/0/392891325557410/1182648556015087
Tech Design URL:
CC:

**Description**:

Adds debug indicator to pixels and makes sure we don't queue an unnecessary number of downloads.

**Steps to test this PR**:
1. Install and monitor downloads.  
1.  Ensure updating the tracker list, etc still works

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
